imgui_impl_vitagl.h

```diff
@@ -14,5 +14,7 @@ IMGUI_API void        ImGui_ImplVitaGL_InvalidateDeviceObjects();
 IMGUI_API bool        ImGui_ImplVitaGL_CreateDeviceObjects();
 
 void ImGui_ImplVitaGL_TouchUsage(bool val);
+void ImGui_ImplVitaGL_FrontTouchpadMode(bool val);
+void ImGui_ImplVitaGL_RearTouch(bool val);
 void ImGui_ImplVitaGL_KeysUsage(bool val);
```

Comment:
I added user functions to control touch behavior.

Reply:
ImGui_ImplVitaGL_FrontTouchpadMode with a bool as argument is not really self-explanatory how should be used and what does it do.